.github/pull_request_template.md

```diff
@@ -2,8 +2,8 @@
 ### Checklist
 
 - Jira Reference (also in PR title):
-- [README](/docs/README.md) updated after adding/changing behaviour of an action
-- Proposed version increment for [release](/docs/README.md#release):
+- [README](https://github.com/Alfresco/alfresco-build-tools/blob/master/docs/README.md) updated after adding/changing behaviour of an action
+- Proposed version increment for [release](https://github.com/Alfresco/alfresco-build-tools/blob/master/docs/README.md#release):
```

Comment:
```suggestion
- [README](../docs/README.md) updated after adding/changing behaviour of an action
- Proposed version increment for [release](../docs/README.md#release):
```

Reply:
tried live in this pr description, still lead to 404 (https://github.com/Alfresco/alfresco-build-tools/docs/README.md#release)

I think magic links works fine when a markdown is rendered, but in this case it's a markdown used as source for PR description so...

Reply:
You're right I thought I used this successfully elsewhere but turns out it's buggy as well.